pkg/segment/writer/segstore.go

```diff
@@ -280,7 +273,11 @@ func (segstore *SegStore) resetSegStore(streamid string, virtualTableName string
 	segstore.LastSegPqids = make(map[string]struct{})
 	segstore.numBlocks = 0
 	segstore.timeCreated = time.Now()
-	segstore.usingSegTree = false
+	if segstore.stbHolder != nil {
+		log.Infof("resetSegStore: Release STB due to reset")
```

Comment:
we can remove this log line, not that helpful

Reply:
@vasukalariya you missed this comment

Reply:
addressed in the latest commit.